.rhcicd/clowdapp-engine.yaml

```diff
@@ -254,6 +254,9 @@ parameters:
 - name: BACKOFFICE_HOST
   description: backoffice URL
   value: backoffice-proxy.apps.ext.spoke.preprod.us-west-2.aws.paas.redhat.com
+- name: BACKOFFICE_SCHEME
+  description: The scheme for the backoffice host's URL
+  value: https
```

Comment:
Probably a more disruptive change, but wouldn't it be better to  use something like BACKOFFICE_URL ?

Also - i find it curious that we are using an external url to talk with BOP - thought it was within the cluster. 

WDYT @gwenneg  ?

Reply:
Done!